lib/league_stat.rb

```diff
@@ -28,6 +28,7 @@ def create_league_stats(games)
   def game_stats_away(game)
     stats_by_team[game.away_team_id][:away_goals] += game.away_goals
     stats_by_team[game.away_team_id][:away_goals_allowed] += game.home_goals
+    stats_by_team[game.away_team_id][:total_games] += 1
```

Comment:
did not want to add total games since i can add home and away wins and losses (already in here) but the methods would get long with argument, so i added another key/value pair here.